src/pages/contributing/translation-program/contributors.tsx

```diff
@@ -97,56 +81,69 @@ const Contributors = ({
 
       <Content>
         <Breadcrumbs slug={location.pathname} />
-        <h1>
+        <ContentHeading as="h1">
           <Translation id="page-contributing-translation-program-contributors-title" />
-        </h1>
-        <h4>
-          <strong>
+        </ContentHeading>
+        <ContentHeading as="h4">
```

Comment:
To go with the comment on the [ContentHeading component](https://github.com/ethereum/ethereum-org-website/pull/9369/files#diff-fea7d7fee9efeaf79d0def83b648b3851e5ae6ef52069d898bcb37b54fb24341R27)

```suggestion
        <ContentHeading
          as="h4"
          fontSize={{ base: "md", md: "xl" }}
          fontWeight={500}
        >
```